### PR TITLE
new file:   packages/nlopt/nlopt.0.7.0/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

File: packages/nlopt/nlopt.0.7.0/opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "dune" {>= "2.9"}
+  "conf-nlopt"
+  "odoc" {with-doc}
+]
+conflicts: [
+  "base-nnp"
+  "ocaml-option-nppchecker"
+]
+url {
+  src: "https://github.com/mkur/nlopt-ocaml/archive/refs/tags/release-0.7.0.tar.gz"
```
Comment (mseri) [inline packages/nlopt/nlopt.0.7.0/opam:29]:
```suggestion
  src: "https://github.com/mkur/nlopt-ocaml/archive/refs/tags/release-0.7.tar.gz"
```

File: packages/nlopt/nlopt.0.7.0/opam [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.9"}
+  "conf-nlopt"
+  "odoc" {with-doc}
+]
+conflicts: [
+  "base-nnp"
+  "ocaml-option-nppchecker"
+]
```
Comment (mseri) [inline packages/nlopt/nlopt.0.7.0/opam:27]:
```suggestion
  "ocaml-options-no-flat-float-array"
]
```

Due to
```
#=== ERROR while compiling nlopt.0.7.0 ========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-options-only-no-flat-float-array.1 | pinned(https://github.com/mkur/nlopt-ocaml/archive/refs/tags/release-0.7.tar.gz)
# path                 ~/.opam/4.14/.opam-switch/build/nlopt.0.7.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p nlopt -j 47 @runtest
# exit-code            1
# env-file             ~/.opam/log/nlopt-7-b18270.env
# output-file          ~/.opam/log/nlopt-7-b18270.out
### output ###
# File "examples/dune", line 2, characters 9-17:
# 2 |   (names tutorial rosenbrock)
#              ^^^^^^^^
# (cd _build/default/examples && ./tutorial.exe)
# Command got signal SEGV.
# File "examples/dune", line 2, characters 18-28:
# 2 |   (names tutorial rosenbrock)
#                       ^^^^^^^^^^
# (cd _build/default/examples && ./rosenbrock.exe)
# Command got signal SEGV.
```

File: packages/nlopt/nlopt.0.7.0/opam [hunk 3]
```diff
@@ -24,6 +24,7 @@ depends: [
 conflicts: [
   "base-nnp"
   "ocaml-option-nppchecker"
```
Comment (edwintorok) [inline packages/nlopt/nlopt.0.7.0/opam:26]: Typo, should it be `ocaml-option-nnpchecker`? Although if nlopt.0.7.0 is supposed to work on OCaml 5.0, then these 2 conflicts should be dropped from this version though.

**Review by edwintorok — APPROVED** [review]

**Comment by UnixJunkie** [comment]
thanks for all the fixes

**Comment by UnixJunkie** [comment]
thanks a lot, it unlocks one of my projects to move to ocaml5 !

**Pull request merged**